client/src/main/java/io/getstream/chat/android/client/parser/adapters/CustomObjectGsonAdapter.kt

```diff
@@ -26,7 +26,7 @@ class CustomObjectGsonAdapter(val gson: Gson, val clazz: Class<*>) : TypeAdapter
                 gson.getAdapter(HashMap::class.java).write(writer, null)
             } else {
                 val result = HashMap<String, Any?>()
-                obj.extraData.map { result[it.key] = it.value }
+                obj.extraData.forEach { result[it.key] = it.value }
```

Comment:
I think this is just `result.putAll(obj.extraData)`? (Which is also `result += obj.extraData`, if you want to be fancy)